### DISCUSSION TITLE
Would anyone like more documentation on manually bootstrapping Git repositories?

### DISCUSSION THREAD

**Discussion by carlosonunez** [fluxcd/flux2#5128] [Proposals]
Hi!

I'm using Flux to manage home automation services that run on a small NUC in my house with GitOps. While the GitHub repository I've connected to is private, I might want to make it public someday and opted to bootstrap Flux manually and disable automatically pushing `gotk-{sync,components}.yaml` into it.

I found the documentation on how to do this lacking. While it wasn't conceptually difficult to figure this out (create an SSH-based `GitRepository` CR with a read-only GitHub deploy key), seeing the response to the [only discussion thread I could find](https://github.com/fluxcd/flux2/discussions/1367) on this being, effectively, "trust us bro, you want the automation" was disappointing.

Is anyone else interested in more documentation on this use case? I'm happy to author the pull request for it if there's enough demand.

Thanks for reading!

**Comment by jferr-tempo** [comment]
I am interested!

**Comment by smheidrich** [comment]
I'm interested as well but for a different reason:

I have a bunch of stuff (Helm charts and standalone k8s manifests) that need to be installed on the cluster in order for Flux to connect to a Git repo at all, but I want these resources to be managed by Flux via GitOps once it has been fully set up. If I want to avoid have "duplicate" Flux _and_ non-Flux manifests for these resources, as far as I can tell the only way to achieve this is to install Flux manually (`flux install [--export]` as mentioned in [a comment](https://github.com/fluxcd/flux2/discussions/1367#discussioncomment-726529) in that thread you linked), apply the manifests of these resources I require, then perform Flux's Git repo bootstrapping manually.
